src/odemis/driver/smaract.py

```diff
@@ -900,6 +900,11 @@ class SA_MC_Event(Structure):
         ("u", SA_MC_EventData),
         ]
 
+    def __str__(self):
+        return "SA_MC_EVenet {type: %s, i32: %s}" % \
+            (MC_5DOF_DLL.event_name.get(self.type),
+             MC_5DOF_DLL.err_code.get(self.i32))
```

Comment:
if the event name or the err_code are unknow, they will be shown as None, which is even less useful than the raw value. => write like `MC_5DOF_DLL.event_name.get(self.type, self.type)`, so that it defaults to showing the raw value.